haskell/compile.bzl

```diff
@@ -614,22 +690,22 @@ def _compile_module(
 
     compile_cmd.add(cmd_args(dependency_modules.reduce("packagedb_deps").keys(), prepend = "--buck2-package-db"))
 
-    dep_file = ctx.actions.declare_output("dep-{}_{}".format(module_name, artifact_suffix)).as_output()
+    dep_file = actions.declare_output("dep-{}_{}".format(module_name, artifact_suffix)).as_output()
 
     tagged_dep_file = abi_tag.tag_artifacts(dep_file)
 
     compile_cmd.add("--buck2-dep", tagged_dep_file)
     compile_cmd.add("--abi-out", outputs[module.hash].as_output())
 
-    ctx.actions.run(
+    actions.run(
         compile_cmd, category = "haskell_compile_" + artifact_suffix.replace("-", "_"), identifier = module_name,
         dep_files = {
             "abi": abi_tag,
-            "packagedb": packagedb_tag,
+            #"packagedb": packagedb_tag,
```

Comment:
was this a duplicate is now a left over comment or does this need to be reinstated?

Reply:
addressed later